tests/test_renderers.py

```diff
@@ -4,6 +4,7 @@
 import re
 from collections import MutableMapping, OrderedDict
 
+import coreapi
```

Comment:
This change will be the one causing the failure. Not sure what’s causing the isort inconsistency.

Reply:
https://travis-ci.org/encode/django-rest-framework/jobs/318806673#L499

It’s report test_schemas too so you’lol need to confirm that, but the test_renderers one is a glitch with isort and Python 2.7 (and possibly our configuration, I’m not sure...🙂)